### ISSUE TITLE
[BUG] Server got stuck during load

### ISSUE THREAD

**Issue by vico93** [ErdbeerbaerLP/DiscordIntegration-Forge#271]
**What happened?**
- I just tried to load my server stack with dcintegration loaded, but it stucks around this lines and dont proceed further:
```
[20:22:29] [Server thread/INFO] []: Loading mod
[20:22:29] [Server thread/WARN] []: SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
[20:22:29] [Server thread/WARN] []: SLF4J: Defaulting to no-operation (NOP) logger implementation
[20:22:29] [Server thread/WARN] []: SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[20:22:30] [Server thread/WARN] []: SLF4J: Failed to load class "org.slf4j.impl.StaticMDCBinder".
[20:22:30] [Server thread/WARN] []: SLF4J: Defaulting to no-operation MDCAdapter implementation.
[20:22:30] [Server thread/WARN] []: SLF4J: See http://www.slf4j.org/codes.html#no_static_mdc_binder for further details.
[20:22:30] [Server thread/WARN] []: [Server thread] INFO JDA - Login Successful!
[20:22:31] [JDA MainWS-ReadThread/WARN] []: [JDA MainWS-ReadThread] INFO WebSocketClient - Connected to WebSocket
```
- It just freezes and i need to kill the process in Task Manager to end it;

**What should happen?**
- The server should continue the loading and be able to have players connecting.

**Steps to reproduce**
- Download Mohist #219 from https://mohistmc.com/download/, run with CustomJava (just try to run Mohist .jar with a JRE with a greater version, like 15.0);
- Place dcintegration-1.1.18-1.12.jar in `mods` folder;
- Configure with the config file i provided (just need the token and channel id) and try to run the mod.

**Version**
- [X] 1.12
- [ ] 1.16

**Platform**
- [ ] Forge
- [ ] Spigot
- [X] Mohist (Forge + Spigot Hybrid)

Mod version: 1.1.18

**Config file**
- https://gist.github.com/vico93/56499bb8701cc8b9d8cc52ed4ec0a880

**Additional info**
-

**Comment by ErdbeerbaerLP** [comment]
Might be fixed with an actually planned backport to 1.12 from the current state
(will start with it as soon as I think the 1.16 is stable enough for release state again)

**Comment by ErdbeerbaerLP** [comment]
Is it fixed? If not just re-open
